core/ir/synthetic/decode.c

```diff
@@ -47,29 +48,33 @@
 byte *
 decode_from_synth(dcontext_t *dcontext, byte *encoded_instr, instr_t *instr)
 {
+    /* Clear the instruction.
```

Comment:
The API requires the user to initialize the instr before calling here, so we should not add overhead clearing it again.

Reply:
Removed initialization.